Exiled.API/Features/Pickups/FirearmPickup.cs

```diff
@@ -0,0 +1,87 @@
+// -----------------------------------------------------------------------
+// <copyright file="FirearmPickup.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Pickups
+{
+    using Exiled.API.Features.Items;
+
+    using InventorySystem.Items.Firearms;
+
+    using BaseFirearm = InventorySystem.Items.Firearms.FirearmPickup;
+
+    /// <summary>
+    /// A wrapper class for Firearm.
+    /// </summary>
+    public class FirearmPickup : Pickup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FirearmPickup"/> class.
+        /// </summary>
+        /// <param name="pickupBase">The base <see cref="BaseFirearm"/> class.</param>
+        public FirearmPickup(BaseFirearm pickupBase)
+            : base(pickupBase)
+        {
+            Base = pickupBase;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="BaseFirearm"/> that this class is encapsulating.
+        /// </summary>
+        public new BaseFirearm Base { get; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the pickup is already distributed.
+        /// </summary>
+        public bool IsDistributed
+        {
+            get => Base.Distributed;
+            set => Base.Distributed = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the <see cref="FirearmStatus"/>.
+        /// </summary>
+        public FirearmStatus Status
+        {
+            get => Base.NetworkStatus;
+            set => Base.NetworkStatus = value;
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating how many ammo have this <see cref="FirearmPickup"/>.
+        /// </summary>
+        public byte Ammo
+        {
+            get => Base.NetworkStatus.Ammo;
+            set => Base.NetworkStatus = new FirearmStatus(value, Base.NetworkStatus.Flags, Base.NetworkStatus.Attachments);
+        }
+
+        /// <summary>
+        /// Gets or sets the <see cref="FirearmStatusFlags"/>.
+        /// </summary>
+        public FirearmStatusFlags Flags
+        {
+            get => Base.NetworkStatus.Flags;
+            set => Base.NetworkStatus = new FirearmStatus(Base.NetworkStatus.Ammo, value, Base.NetworkStatus.Attachments);
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the Attachement code have this <see cref="FirearmPickup"/>.
+        /// </summary>
+        public uint Attachments
+        {
+            get => Base.NetworkStatus.Attachments;
+            set => Base.NetworkStatus = new FirearmStatus(Base.NetworkStatus.Ammo, Base.NetworkStatus.Flags, value);
+        }
+
+        /// <summary>
+        /// Returns the AmmoPickup in a human readable format.
```

Comment:
You forgot to change this in 8a190ff70414564e41a83f2566a4a9d534ff83d5 you change only `returns` text

Reply:
Oups dumb moment